test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -931,15 +930,17 @@ def forward(self, x):
         def create_args():
             return (torch.rand(5, 2, 2),)
 
-        def create_pytorch_only_extra_kwargs():
```

Comment:
How was this merged last time ...

Reply:
copy&paste not noticed during coding/review